src/main/java/svc/managers/CitationManager.java

```diff
@@ -67,4 +73,24 @@ public Citation GetCitationById(Long citationId) {
 		}
 		return citations;
 	}
+	
+	public String getCitationSMS(Citation citation){
```

Comment:
Nothing SMS related should be in a manager that's managing Citations. SMSManager could relay on CitationManager but not the other way around. This will dig us into a hole and make future enhancements less flexible.